tests/vite-app/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "vite-app",
+  "private": true,
+  "version": "0.0.2",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "tokengen": "tsc util/generateExampleTokens.ts && node util/generateExampleTokens.js",
+    "cypress:run": "cypress run",
+    "cypress:run:preview": "cypress run --config baseUrl=http://localhost:4173/",
+    "cypress:open": "cypress open",
+    "cypress:open:webpack": "DEBUG=cypress:webpack cypress open",
+    "cypress:open:preview": "cypress open --config baseUrl=http://localhost:4173/"
+  },
+  "dependencies": {
+    "aws-jwt-verify": "file:../../aws-jwt-verify.tgz"
```

Comment:
Let's sync the way we do this across all the tests we have, think for others we don;t put it in package.json but do `npm i --force --no-package-lock` instead before each test run